tap_recharge/client.py

```diff
@@ -187,7 +190,7 @@ def request(self, method, path=None, url=None, **kwargs):
             LOGGER.error('{}'.format(err))
             raise Exception(err)
 
-        return response_json
+        return (response_json, response.links)
```

Comment:
Note: python will implicitly return a tuple when you have comma separated return values. So the parens here are not required